tests/baselines/reference/importTypeLocal.types

```diff
@@ -66,9 +66,9 @@ export const x: import("./foo") = { x: 0, y: 0 };
 >0 : 0
 
 export let y: import("./foo2").Bar.I = { a: "", b: 0 };
->y : Bar.I
+>y : import("tests/cases/conformance/types/import/foo2").Bar.I

```

Comment:
Shouldn't this be import("./foo") instead of this whole path?

Reply:
Not in the `types` baselines - they have no enclosing declaration specified for their type writer, so they get the whole path.